README.md

```diff
@@ -0,0 +1,140 @@
+## SDK Features
+
+This repository contains snippets for many Temporal SDK features written in different Temporal SDK languages. This also
+contains a runner and language-specific harnesses to confirm feature behavior across versions.
+
+These SDK features serve several purposes:
+
+* Ensure parity across SDKs by having same-feature snippets adjacent to one another
+* Confirm feature behavior across SDK versions
+* Confirm history across SDK versions
+* Document features in different SDKs
+* Easy-to-use environment for writing quick workflows in all languages/versions
+
+## Building
+
+With latest [Go](https://golang.org/) installed, run:
+
+    go build
+
+## Running
+
+Prerequisites:
+
+* [Go](https://golang.org/) 1.17+
+* [JDK](https://adoptium.net/?variant=openjdk11&jvmVariant=hotspot) 11+
```

Comment:
Do we use any of the >8 Java lang features?
If not, it's better to downgrade to JDK 8, because it's still our supported version by the SDK.

Reply:
> Do we use any of the >8 Java lang features?

Yup, that's one of the really nice things about testing frameworks like this is that you can choose whatever language you want because you don't have compat concerns like a library might.

Reply:
Btw, not sure if I made it clear from the description, but there are 4 other PRs that have the actual impl, and one of them is the Java harness. Review on those very welcome too.